devicetypes/smartthings/zigbee-motion-temp-humidity-sensor.src/zigbee-motion-temp-humidity-sensor.groovy

```diff
@@ -0,0 +1,284 @@
+/*
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "Zigbee Motion/Temp/Humidity Sensor", namespace: "smartthings", author: "SmartThings", mnmn: "SmartThings", vid: "generic-motion-6") {
```

Comment:
`generic-motion-6` also includes the "Illuminance Measurement" capability. That means we'd have an empty tile in the new app, which is generally a no-no. I can see about creating a "generic-motion-8" or similar this weekend, but we don't want to use this one.

Reply:
yeah, that's the closest I knew of. Let me know if you create a better fit.

Reply:
I don't see Illuminance Measurement in the new app - did something change?

Reply:
The new app hides the capability if it’s null.

Reply:
If that's the case then we don't need to generate new metadata I guess. This would be good to go.